src/components/CheckoutFunnel/PersonalInformation/PersonalInformation.tsx

```diff
@@ -217,43 +247,79 @@ export class PersonalInformation extends React.Component<Props, State> {
     );
   };
 
+  async onSubmit() {
+    await this.setEmail();
+    await this.setShippingAddress();
+    // Show server errors if needed
+    // this.props.history.push(PAYMENT_URL);
```

Comment:
We should let this uncommented so that we can still go through the flow without manually having to update the URL.